Common/Securities/Volatility/IndicatorVolatilityModel.cs

```diff
@@ -80,6 +78,26 @@ public void Update(Security security, BaseData data)
             }
         }
 
+        /// <summary>
+        /// Applies a dividend to the portfolio
+        /// </summary>
+        /// <param name="dividend">The dividend to be applied</param>
+        /// <param name="liveMode">True if live mode, false for backtest</param>
+        /// <param name="mode">The <see cref="DataNormalizationMode"/> for this security</param>
+        public void ApplyDividend(Dividend dividend, bool liveMode, DataNormalizationMode mode)
```

Comment:
This is relying on the owner of the indicator to handle the split-dividend events. If the stock splits in live trading it will still suffer a discontinuity (indicators don't currently automatically re-adjust post-split). 

Given we don't have indicators automatically re-initializing shouldn't we handle it for them here?

Reply:
In order to handle it, we will need to implement a "refresh" logic to each indicator similar to what I have done to `RelativeStandardDeviationVolatilityModel` and `StandardDeviationOfReturnsVolatilityModel`. 
Alternatively, we could ensure that indicators always receive adjusted data (https://github.com/QuantConnect/Lean/issues/383) at least in this case.